docusaurus/docs/themes/transportation/roads.mdx

```diff
@@ -0,0 +1,606 @@
+import ExampleTurnRestriction1Connector2 from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-01-connector2.yaml';
+import ExampleTurnRestriction1Connector3 from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-01-connector2.yaml';
+import ExampleTurnRestriction2Source from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-02-source.yaml';
+import ExampleTurnRestriction2Target from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-02-target.yaml';
+import ExampleTurnRestriction2Via from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-02-via.yaml';
+import ExampleTurnRestriction2Connector1 from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-02-connector1.yaml';
+import ExampleTurnRestriction2Connector2 from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-02-connector2.yaml';
+
+
+# Roads
+
+In the Overture Transportation theme, a road is any kind of road,
+street, or path, including a dedicated path for walking or cycling
+(but excluding a railway). Road segments comprise the majority of
+ground-based transportation segments. Roads are modeled using
+[Segment](/reference/transportation/segment) features with the `subType`
+property value set to the value `road`.
+
+## Granularity and geometry
+
+Road segments support modeling the road network at a range of
+granularities. For example, a single road segment can represent:
+
+- A bidirectional street including all of its lanes and sidewalks.
+- A sidewalk by itself.
+- A one-way street or one direction of a dual carriageway.
+- A single lane or single section of a multi-lane highway.
+- A dedicated cycleway.
+- A hiking path.
+
+# Centerline and width
+
+A road segment's geometry represents the centerline of the section of
+road being modeled by the segment, with exactly half of the road's width
+occurring on each side of the segment at any given point along the
+segment geometry.
+
+<div style={{textAlign: 'center'}}>
+
+<figure>
+
+![Road segment geometry as centerline](/img/transportation/centerline.svg)
+
+<figcaption>
+
+*A road segment represents the centerline of a path, with half the width on each side.*
+
+</figcaption>
+</figure>
+
+</div>
+
+If the exact width of a road segment is known, it will be specified in
+the segment's `width` property (as a quantity in meters). If the exact
+width is not known, the road geometry still represents the centerline,
+but the data consumer will need to estimate the width if it is needed.
+
+## Basic Properties
+
+### Road class
+
+The `road.class` property of a road specifies its general purpose of use
+and its relative importance within the transportation network. The
+`road.class` property also helps establish reasonable default values.
```

Comment:
We don't currently give a lot of 'reasonable default values', but is it worth stating in the segments model that a segment is expected to have a single default value for the entire segment? It seems implied as a reason to split segments on `road.class` changes, but could be a useful way to help frame where we pick to split segments.

Reply:
Not an answer, but: I think we might want to question the premise of splitting on `class`.